stdlib/cmake/modules/AddSwiftStdlib.cmake

```diff
@@ -288,6 +288,18 @@ function(_add_target_variant_c_compile_flags)
     endif()
   endif()
 
+  # Avoid the need for the FP16 truncation and extension routines from
+  # compiler-rt by assuming that we have hardware capable of performing
+  # half-precision floating point conversions.  This was announced circa 2007,
+  # and is standard with ~Nahelem (~2013).
```

Comment:
```suggestion
  # and is standard with ~Haswell (~2013).
```

IIUC

Reply:
Yes, sorry about that!  Fixed that locally.